app/components/e-input.md

```diff
@@ -8,7 +6,7 @@
     <e-input v-model="demo" name="demo" placeholder="enter text..."/> 
     
     <!-- demo only -->
-    <div v-if="demo"><br/>
+    <div v-if="demo" class="spacing--top-15">
```

Comment:
I think this is not working properly. Have I done a mistake?

Reply:
Bug in the mixin. Fixed it.